velero-csi-plugin/csi_restorer_test.go

```diff
@@ -96,26 +98,21 @@ func TestResetPVCAnnotations(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			resetPVCAnnotations(&tc.pvc, tc.preserveAnnotations)
-			if tc.pvc.Annotations == nil {
-				t.Errorf("%s failed, unexpected PVC annotations returned, Want: non-nil; Got: nil", tc.name)
-			}
-			if tc.expectNonEmptyAnnotations && len(tc.pvc.Annotations) != 0 {
-				t.Errorf("%s failed, unexpected count of PVC annotations returned, Want: 0; Got: %d", tc.name, len(tc.pvc.Annotations))
+
+			assert.NotNil(t, tc.pvc.Annotations, "unexpected PVC annotations returned, Want: non-nil; Got: nil")
+
+			if tc.expectNonEmptyAnnotations {
+				assert.Emptyf(t, tc.pvc.Annotations, "unexpected count of PVC annotations returned, Want: 0; Got: %d", len(tc.pvc.Annotations))
```

Comment:
something seems reversed here - when tc.expect**Non**EmptyAnnotations is true, we assert that tc.pvc.Annotations is empty?

Reply:
That was confusing. I updated the tests to make it more clear. PTAL.